codeowner-scan.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
```

Comment:
This is nice! Would you mind adding a comment in here that provides just a little more guidance? Maybe a brief description of the command line arg it takes, whether or not you need to be authenticated to gh api, etc? Thanks!